kumascript/macros/APIListAlpha.ejs

```diff
@@ -81,17 +81,17 @@ for (var i = 0; i < numLetters; i++) {
 
             // add badges if needed
 
-            if (containsTag(tags, "Non-standard") || containsTag(tags, "Non standard")) {
+            if (pages[p].status?.includes("non-standard") || containsTag(tags, "Non-standard") || containsTag(tags, "Non standard")) {
```

Comment:
Note, some tests failed, I think because `includes` was called on an undefined object. So I made this a conditional comparison that defaults to undefined (falsy) for that case.

Reply:
See https://github.com/mdn/yari/pull/7828#issuecomment-1358673590 though - you should not have to do this. As far as I could see the tests passed after my https://github.com/mdn/yari/pull/7828/commits/b3701c5262494283df3b60ec98864cb7633ae441.